tests/jeos/prepare_firstboot.pm

```diff
@@ -45,7 +45,7 @@ sub run {
             # Handle default credentials for ssh login
             # On SLE we use an image preprocessed by openQA where the default
             # $testapi::password was set
-            $testapi::password = $default_password;
+            local $testapi::password = $default_password;
         }
         # 'root-ssh' console will wait for SUT to be reachable from ssh
         select_console('root-ssh');
```

Comment:
I guess it should just reset `$testapi::password` after the `select_console`?

Reply:
It should have https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/jeos/prepare_firstboot.pm#L78

Reply:
alright, even the original `$testapi::password` is changed outside this module.
@ggardet, is it possible that the password somewhere in the testapi code is changed in RPi tests?

Reply:
Default password is `linux` on JeOS images.

Reply:
I have done a PoC, https://openqa.opensuse.org/tests/4457380#step/firstrun/14 shows that `$testapi::password` is set to `linux`. In the PoC run I have changed it, the UI test object passed. Later the password was restored to `linux` (default) hence the test fails in another point.
If we keep `linux` as default, I would rather skip this test object by setting `WIZARD_SKIP_USER` in community jeos RPi test runs